bee-node/src/plugins/dashboard/config.rs

```diff
@@ -74,16 +76,44 @@ impl DashboardConfigBuilder {
     }
 
     pub fn finish(self) -> DashboardConfig {
+        let multi_addr = self
+            .bind_addresses
+            .unwrap_or_else(|| DEFAULT_BIND_ADDRESSES.parse().unwrap());
```

Comment:
add a small comment why `unwrap`ing is safe. I suggest:
"We made sure that the default value is valid and therefore parseable.". Alternatively you could use `expect("error parsing default bind addresses")`.